cpp/test/Ice/timeout/AllTests.cpp

```diff
@@ -88,47 +88,6 @@ allTestsWithController(Test::TestHelper* helper, const ControllerPrx& controller
     }
     cout << "ok" << endl;
 
-    // The sequence needs to be large enough to fill the write/recv buffers
-    ByteSeq seq(2000000);
-
-    cout << "testing connection timeout... " << flush;
-    {
-        //
-        // Expect TimeoutException.
-        //
-        TimeoutPrx to = timeout->ice_timeout(250);
-        connect(to);
-        controller->holdAdapter(-1);
-        try
-        {
-            to->sendData(seq);
```

Comment:
Do we have a similar test for the new idle timeout? I suppose that if the connection writes are stuck in both directions the connection is not killed?

Reply:
That's correct.  With this update, a write can get stuck forever. We actually have exactly the same semantics with IceRPC: with IceRPC, the only way to interrupt a stuck write with the ice protocol is to abort the connection.

The assumption here (and in IceRPC) is if the connection is sick and writes are stuck, the idle check will abort the connection. We should indeed add a test to verify the idle check can indeed abort a connection when this connection is stuck writing.

Reply:
The removed portion of the metrics test (as I understand it) was generating connection errors by having write hang because of a low "-t" (500 ms). Since this feature (-t) is removed by this PR, this no longer works.